site/data/refDataApi.ts

```diff
@@ -151,13 +151,18 @@ export const fetchServiceStops = async (input: FetchServiceStops) => {
 
 interface FetchOperatorsInput {
     adminAreaCodes: string[];
+    dataSource?: Modes;
 }
 
 export const fetchOperators = async (input: FetchOperatorsInput) => {
```

Comment:
We need to clear the table of operators when mode changes

Reply:
No need clear the table as this has been verified with steve